site/app/views/forum/ForumThreadView.php

```diff
@@ -515,13 +524,18 @@ public function generatePostList($currentThread, $posts, $unviewed_posts, $curre
 
                         $post["author_user_group"] = $author_user_groups_map[$post["author_user_id"]];
 
+                        $boolLiked = true;
```

Comment:
unnecessary?

Reply:
it needed a type for php liner so I did that to fix that error

Reply:
What was the linting error?

Reply:
I think it was something about not being initialized or not being declared anywhere.

Reply:
Can you remove it and generate that linting error? We definitely should not need that line.